### ISSUE TITLE
BetterTouchTool captures more mouse buttons [other]

### ISSUE THREAD

**Issue by asoksevil** [noah-nuebling/Mac-Mouse-Fix#405]
Hi,

Awesome app however I wonder why it can't capture some buttons that BTT can? For instance my mouse middle button can move to the left or right.

**Comment by ksuuk** [comment]
@asoksevil Yes, unfortunately, there is no left/right tilt action of the scroll wheel, but this is a much-requested feature.
